websocket-sink/src/main/java/org/springframework/cloud/stream/module/websocket/sink/WebsocketSinkProperties.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2014-15 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.springframework.cloud.stream.module.websocket.sink;
+
+import io.netty.handler.logging.LogLevel;
+import lombok.*;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * command line arguments available for {@link WebsocketSink}
+ *
+ * @author Oliver Moser
+ */
+@Data
+@ConfigurationProperties(prefix = "ws")
```

Comment:
We can possibly avoid having `prefix` for the module properties. Since we always pass the module properties with clear distinction on which module gets what properties, the prefix is an overhead I believe.